datasets/modeled/transport/dataset.json5

```diff
@@ -1,59 +1,94 @@
 {
-  dataset_id: "tempo_conus_2022",
+  dataset_id: "decarb_2023_transport",
   dataset_type: "modeled",
-  data_source: "tempo",
-  sector_description: "transportation",
-  description: "CONUS 2022 TEMPO county-level results for PEV charging.",
+  data_source: "tempo evix polaris",
+  sector_description: "electric load associated with light, medium, and heavy-duty road transportation",
+  description: "DECARB 6 results from TEMPO, EVI-X, and POLARIS (Nov 2023) with profiles using 2018 as the weather year",
   origin_creator: "Arthur Yip",
   origin_organization: "NREL",
   origin_contributors: [
     "Arthur Yip",
+    "Brennan Borlaug"
```

Comment:
There are probably more collaborators that should be credited? TEMPO and POLARIS teams???